### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by syershov**

### CODE REVIEW

File: coding/internal/xmlparser.hpp [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+
+#define XML_STATIC
 #include "3party/expat/expat_impl.h"
 
-#include "base/logging.hpp"
+#if defined(__clang__)
+#pragma clang diagnostic pop
+#endif
 
+#include "base/logging.hpp"
```
Comment (ygorshenin) [inline coding/internal/xmlparser.hpp]: This should be before 3party, isn't it?

Comment (syershov) [inline coding/internal/xmlparser.hpp]: fixed

**Comment by syershov** [comment]
@ygorshenin PTAL

**Comment by ygorshenin** [comment]
LGTM

**Pull request merged**